schemamd/render.go

```diff
@@ -225,6 +225,13 @@ nameLoop:
 			}
 		} else if childAtt, ok := block.Attributes[n]; ok {
 			for i, gf := range groupFilters {
+				if strings.ToLower(n) == "id" && childAtt.Description == "" {
+					if gf.topLevelTitle == "Read-Only" {
```

Comment:
Did you verify this? Because I think this won't match anything: I believe the string is `### Read-Only`.

It would be even better if the elements of `groupFilters` above existed as variables themselves, and so you could directly match that and avoid bugs out of string mismatches (now and in the future).

Reply:
That's correct, I have it working with that change (see below).